src/libs/virtualKeyboard.js

```diff
@@ -0,0 +1,18 @@
+import _ from 'underscore';
```

Comment:
I thought about this some more, and I think it's a bad idea to create this library with only the web implementation when we can easily add a native implementation. That way, nobody will be surprised if they later attempt to use this library in another context and find that it always returns `null` in the native platforms. To implement this, simply do the following:

1. Move this file to `src/libs/VirtualKeyboard/index.js`
1. Create a new file called `src/libs/VirtualKeyboard/index.native.js`, and give it the following content:

```javascript
let isVirtualKeyboardOpen = false;

Keyboard.addListener(
    'keyboardDidShow',
    () => {
      isVirtualKeyboardOpen = true;
    },
);

Keyboard.addListener(
    'keyboardDidHide',
    () => {
        isVirtualKeyboardOpen = false;
    },
);

/**
 * Is the virtual keyboard open?
 *
 * Note – the web equivalent of this function may return null.
 *
 * @returns {Boolean}
 */
function isOpen() {
    return isVirtualKeyboardOpen;
};

export {
    // eslint-disable-next-line import/prefer-default-export
    isOpen,
};
```

Reply:
yes, it will be helpful in the case of a tablet with a physical keyboard. using a native app.

Reply:
it'd done now @roryabraham